libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -147,9 +151,9 @@
     private LoginOptions loginOptions;
     private final Class<? extends Activity> mainActivityClass;
     private Class<? extends Activity> loginActivityClass = LoginActivity.class;
-    private Class<? extends PasscodeActivity> passcodeActivityClass = PasscodeActivity.class;
+    private Class<? extends ScreenLockActivity> screenLockActivityClass = ScreenLockActivity.class;
```

Comment:
Remove this. Now that it's handled by the OS, we don't want apps to customize this anymore. They can tweak the colors and branding  by overriding it in XML. The only reason we allowed `PasscodeActivity` to be overridden was  for legacy reasons  specific  to `SalesforceApp`.